tests/conftest.py

```diff
@@ -39,13 +42,13 @@ def webserver():
 
 
 @pytest.fixture
-def browser(foxpuppet):
+def browser(foxpuppet: FoxPuppet) -> BrowserWindow:
     """First Firefox browser window opened."""
     return foxpuppet.browser
 
 
 @pytest.fixture
-def firefox_options(firefox_options):
+def firefox_options(firefox_options: Any) -> Any:
```

Comment:
You can use this, `from selenium.webdriver.firefox.options import Options as FirefoxOptions`